### ISSUE TITLE
Terraform Module is not up to date

### ISSUE THREAD

**Issue by Viriathus1** [RaJiska/terraform-aws-fck-nat#37]
The current terraform module is sufficiently out of step with the documentation and work on the main branch.

In order to use it we must use the deprecated input variables like update_route_table and route_table_id along with other input variables not being available like use_ssh. 

Is it possible to get an estimated time on the next release?

**Comment by leo-the-nardo** [comment]
i had the same issue, i was able to use the new version cloning this repo and importing the local module as workaround. : 
```
module "fck-nat" {

  source    = "./modules/fck-nat"
  name      = "fck-nat"
  vpc_id    = aws_vpc.cluster_vpc.id
  subnet_id = aws_subnet.public_subnet_az1.id
  # ha_mode              = true                 # Enables high-availability mode
  # use_cloudwatch_agent = true                 # Enables Cloudwatch agent and have metrics reported

  update_route_table = true
  route_tables_ids = {
    az1 = aws_route_table.router.id
    az2 = aws_route_table.router.id
  }
  use_ssh = true
  ssh_cidr_blocks = {
    ipv4 = [var.ssh_cidr]
    ipv6 = []
  }
  eip_allocation_ids                 = [aws_eip.router_eip.id]
}
``

**Comment by genghis-tuan** [comment]
Version 1.2.0 also doesn't handle x86_64 instance types in the regex which determines the `local.is_arm`. This causes the Terraform to fail when instance_type is, for example `t2.micro` with `Call to function "regex" failed: pattern did not match any part of the given string.`. This  is fixed in the main branch. Would it be possible to get a release?

**Comment by Viriathus1** [comment]
@leo-the-nardo yes, may have to use that workaround.

@genghis-tuan Second for this issue as well.

**Comment by RaJiska** [comment]
@Viriathus1 @leo-the-nardo @genghis-tuan Just published a new version v1.3.0 which should include all the latest changes.

Apologies for the delay. Lately I have been experimenting a lot with true-HA mechanisms and wanted to see if I could get it done before releasing a new version (which I couldn't achieve).

**Comment by Viriathus1** [comment]
@RaJiska thanks mate. On another note, currently getting a warning for aws_iam_role as a deprecated argument. The suggestion is to use aws_iam_role_policy. If this is not already on your radar, I'll take a look at it when I have time.

**Comment by bobsut** [comment]
@Viriathus1 - Reported in #41 and fixed in #42
